eventmesh-examples/src/main/resources/log4j2.xml

```diff
@@ -1,19 +1,19 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!--
-  ~ Licensed to the Apache Software Foundation (ASF) under one or more
-  ~ contributor license agreements.  See the NOTICE file distributed with
-  ~ this work for additional information regarding copyright ownership.
-  ~ The ASF licenses this file to You under the Apache License, Version 2.0
-  ~ (the "License"); you may not use this file except in compliance with
-  ~ the License.  You may obtain a copy of the License at
-  ~
-  ~     http://www.apache.org/licenses/LICENSE-2.0
-  ~
-  ~ Unless required by applicable law or agreed to in writing, software
-  ~ distributed under the License is distributed on an "AS IS" BASIS,
-  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-  ~ See the License for the specific language governing permissions and
-  ~ limitations under the License.
+   Licensed to the Apache Software Foundation (ASF) under one or more
+   contributor license agreements.  See the NOTICE file distributed with
```

Comment:
The `~ ` should be removed, not just the `~`, to maintain the correct two-space indentation.

Reply:
> The `~ ` should be removed, not just the `~`, to maintain the correct two-space indentation.

I will do later

Reply:
Most of `log4j.xml` files has been removed in https://github.com/apache/eventmesh/pull/4719, and a `checkstyle-header-xml.txt` header file has been added in https://github.com/apache/eventmesh/pull/4827. 

It seems the xml header file with `~` included is the default header applied by apache/skywalking-eyes action. Do you think we still need to remove the `~` in the header?

Reply:
@pchengma @mxsm We have an unresolved conversation here~ Please take a look.

Reply:
I think both styles are OK. If `~ ` is auto-generated, it is fine to keep it. Nevertheless, some projects may not use `~ `, for example [fineract](https://github.com/apache/fineract/blob/53f4864980228740fb7e1adddaca476aa1539856/config/checkstyle/checkstyle.xml).

Reply:
Yes, the `apache/skywalking-eyes` action is considered valid for both license headers with and without `~`. However it adds `~` when auto-generating. Considering that all xml license headers in EventMesh now have `~`, we can unify having `~`.